### ISSUE TITLE
True tile rendering?

### ISSUE THREAD

**Issue by Wynadorn** [117HD/RLHD#317]
**Is your feature request related to a problem? Please describe.**
I have true tiles enabled and while playing I noticed most of the time I look at the tile markers and not the models.
And then I wondered if it would be feasible to just render the models on their true tiles.
It would deftly make the game feel more responsive.

**Describe the solution you'd like**
Would it be feasible to render models on their true tile?

**Describe alternatives you've considered**
Is it possible to do this without breaking animations?
Can this be applied to more than just the player model?

**Additional context**
Currently when moving the player they lag behind their true tile.
This will result in the player "chasing" their true location.
I think this is caused by a delay before starting the move animation.
I think I'd be really cool if we could make these be more in-sync.
![image](https://github.com/117HD/RLHD/assets/4095127/597b982a-b1ac-48d7-b4f2-d7e28ba29e6f)

**Comment by Mark7625** [comment]
I looked at this a while back, moving the actual player is not a good idea as people are used to how it is. Another issue with moving the player is maybe Projectiles and other things would have to be moved also and could be confusing when bossing. I did look into showing maybe a ghost version of your player on the actual tile but, it was kinda confusing to play with

**Comment by Wynadorn** [comment]
I suppose this would have to be implemented fairly low-level so that projectiles, 2d graphics (hit splats, overheads, etc.) all take this change into account. Even then I think it'll a experimental feature at most.

However, you could also argue that Jagex's own implementation isn't accurate either. Especially if you consider animation stalls.

**Comment by Mark7625** [comment]
This is rlly outside the scope of this plugin, While what you suggest could be done it would just confuse people, and would have to do a lot of work as I outlined above, Sorry
